benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadWorker.java

```diff
@@ -116,6 +116,12 @@ public static void main(String[] args) throws Exception {
     LoadWorker loadWorker = new LoadWorker(driverPort, serverPort);
     loadWorker.start();
     loadWorker.driverServer.awaitTermination();
+    log.log(Level.INFO, "DriverServer has terminated.");
+
+    // Allow enough time for quitWorker to deliver OK status to the driver.
+    // Also make sure this process actually exits (see #1685).
+    Thread.sleep(3000);
+    System.exit(0);
```

Comment:
And since I can't comment on other lines (thanks github):

On line 58, that constructor should be replaced with 

https://github.com/netty/netty/blob/4.1/transport/src/main/java/io/netty/channel/nio/NioEventLoopGroup.java#L55


that one, which provides a ThreadFactory that makes daemon threads.  The the example in  GrpcUtil.